Makefile

```diff
@@ -85,6 +85,11 @@ ifeq ($(WITH_ROLLINGDB),1)
 override SIM_ARGS += --with-rollingdb
 endif
 
+# run with disable all db
+ifeq ($(WITH_DISABLEDB),1)
```

Comment:
`WITH_DISABLEDB` seems weird. How about changing it to `DISABLE_ALWAYSDB`?

Reply:
ok